depinject/config.go

```diff
@@ -48,6 +48,51 @@ func provide(ctr *container, key *moduleKey, providers []interface{}) error {
 	return nil
 }
 
+// BindInterface defines a container configuration for an explicit interface binding of inTypeName to outTypeName
+// in global scope, for example,
+//
+// BindInterface(
+//	"github.com/cosmos/cosmos-sdk/depinject_test/depinject_test.Duck",
+//	"github.com/cosmos/cosmos-sdk/depinject_test/depinject_test.Canvasback")
+//
```

Comment:
I don't find this style of `start sentence -> code sample -> finish sentence` very readable. It would be better IMHO to write a whole sentence, then a code sample, then more sentences (if needed). It's okay for now and I can follow but it might be hard for readers